aeron-driver/src/main/java/io/aeron/driver/media/SendChannelEndpoint.java

```diff
@@ -734,13 +755,20 @@ else if (datagramChannel.isOpen())
                 buffer.position(position);
                 channelEndpoint.sendHook(buffer, destination);
                 bytesSent = datagramChannel.send(buffer, destination);
+
+                if (bytesSent < bytesToSend)
+                {
+                    transportShortSends.incrementRelease();
+                }
             }
         }
         catch (final PortUnreachableException ignore)
         {
+            transportShortSends.incrementRelease();
```

Comment:
I'm wondering if the transport short sends should exclude these and have each as separate counters.  I think the transport short sends should be just the ones seen on line 761, which should be analogous to EAGAIN/EWOULDBLOCK.  I think this should be a separate counter which would be analogous to ECONNREFUSED on the C side.